desktop/app/lib/settings/index.js

```diff
@@ -33,10 +33,6 @@ Settings.prototype.isDebug = function () {
  * If no default setting then fall back to false
  */
 Settings.prototype.getSetting = function ( setting ) {
-	if ( process.env.WP_DESKTOP_BASE_URL && setting === constants.LAST_LOCATION ) {
```

Comment:
@noahtallen This piece of logic actually implies that the URL _isn't_ hard-coded if the base URL override is injected via the `WP_DESKTOP_BASE_URL` environment variable (similar to how we handle using localhost for development purposes below).

This works, which is great, but I'm not sure why it works, which makes me a tiny bit uneasy.